### PR TITLE
Create Leadership Section

### LINKED ISSUE #78: Add Manager Role Profiles to Website
It would be nice to include the managers to the website with links to our GitHub Repos.

Thinking maybe a profile underneath the role.

### PR DESCRIPTION
**Pull request by kjaymiller**
resolves #78

Adds the leadership team to the about page.

### CODE REVIEW

**Comment by kjaymiller** [comment]
Given that your information is being presented on this page I will not merge this until I have approvals from both of you.

cc: @oleksis @VeldaKiara

File: about.md
Comment (oleksis) [inline about.md:49]: The `src` attribute for GitHub Avatar can be more short like `http://github.com/oleksis.png`

**Review by oleksis — APPROVED** [review]

**Review by VeldaKiara — APPROVED** [review]

File: about.md
Comment (VeldaKiara) [reply about.md:49]: I am okay with the changes

**Pull request merged**
